pkgs/servers/webdav-server-rs/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, fetchFromGitHub, rustPlatform, libtirpc, pam, rpcsvc-proto }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "webdav-server-rs";
+  # The v0.4.0 tag cannot build.  So we use the 547602e commit.
```

Comment:
Sounds like an upstream bug. Should we file an issue and get this fixed?

Reply:
The Cargo.lock file in v0.4.0 is outdated.  It's resolved in later commit.  So it's not a bug worth reporting.